modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -3128,6 +3132,19 @@ public void testDeleteObjectEntry() throws Exception {
 
 		_objectEntryLocalService.deleteObjectEntry(objectEntry1);
```

Comment:
@DaniSzimko @ccorreagg could you remove this row for us? I just cheked it doesn't make sense.

`_objectEntryLocalService.deleteObjectEntry(objectEntry1);`